modules/serverless-kafka-sink-security-config.adoc

```diff
@@ -0,0 +1,91 @@
+// Module is included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-sink-security-config_{context}"]
+= Configuring security for Kafka sinks
+
```

Comment:
Need [role="_abstract"] and abstract for the module per https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

Reply:
The last time I asked @kalexand-rh about using `[role="_abstract"]` the guidance AFAIK was that we don't need to use it, so it's currently not used anywhere in our docs. Kathryn, should I open a Jira for serverless to update all the abstracts to use this, or is it still not required?

Reply:
This might be just for OpenShift (I'm new to peer reviewing) :)

Reply:
`[role="_abstract"]` is only for PV2 - it's been removed from the Jupiter guidelines, which OpenShift uses. RHACS is the only content in the OCP repo that requires that role. (The additional resource one is required.)